### ISSUE TITLE
Virtual keyboard is not working when used UWP app on Kiosk mode

### ISSUE THREAD

**Issue by Suresh-BabuLV** [MicrosoftEdge/WebView2Feedback#2694]
**Description**
We have implemented one UWP app that app used the webview2 version 1.0.1264.42 and Microsoft.UI.Xaml version Latest stable 2.8.0 NuGet package added in this project

When we use this app on Kiosk mode with touch panel not enable the Virtual keyboard to type anything from this app due to updated webview2 runtime version 104.0.1293.54 but previously working on webview2 runtime version 100.0.1185.29.

**Version**
webview2 version 1.0.1264.42.
webview2 runtime version 104.0.1293.54
Microsoft.UI.Xaml version Latest stable 2.8.0
[TestVirtualKeyboardApp.zip](https://github.com/MicrosoftEdge/WebView2Feedback/files/9364324/TestVirtualKeyboardApp.zip)

UWP app
Windows 10.

I have attached the reference project here if any modification required please guide where i can modify to work the virtual keyboard on touch windows OS.

Please guide and suggest to working webview2 runtime version.
Thanks in Advance.

Regards,
Suresh.

[AB#41312202](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/41312202)

**Comment by Suresh-BabuLV** [comment]
Hi @victorthoang, @maurawinstanley.

Great Thanks for processing further.

“This is broken functionality – not sure what caused this breakage – Virtual keyboard was working fine in “webview2 runtime version 100.0.1185.29”  and the functionality is broken once upgraded from this version – right now we have reverted to webview2 runtime version 100.0.1185.29 to make sure users have access to Keyboard – appreciate if you could look into what broke this functionality and will help if the next release of WebView2 can address this virtual keyboard as this plays a key role on Surface Pro that we use and also Kiosks ”

Thanks in Advance.

I would be appreciate to the provide valid solution ASAP.


Regards,
Suresh.

**Comment by Suresh-BabuLV** [comment]
@maurawinstanley, Thanks for working on this issue, I request you any update on this one – it is been quite for 2 weeks now – appreciate if someone could update if this broken functionality could be restored – Thank you”


Regards,
Suresh.

**Comment by victorthoang** [comment]
@Suresh-BabuLV, thank you for your patience, we had some resource issues (partly on me) and got it sorted out now. Will make sure that we have someone take a look at this soon.

**Comment by Suresh-BabuLV** [comment]
@victorthoang, Great Thanks, We are not able to release the app due to this issue. It would be great help me to quick resolve for this issue ASAP.
I am awaiting your response is resolved this issue. Kindly Help me ASAP.

Regards,
Suresh.

**Comment by victorthoang** [comment]
@tofuandeve should be able to take a look at your request. Will correspond on the situation.

**Comment by victorthoang** [comment]
@Suresh-BabuLV we are currently investigating and prioritizing our backlog for this issue. Will provide an update/eta soon.

**Comment by Suresh-BabuLV** [comment]
@victorthoang, Great Thanks for taking further action. It would be great to help us to know the status of the issue.

@tofuandeve,Thanks, Please update me on this issue. It seems tracked the issue

Regards,
Suresh.

**Comment by maneevannan** [comment]
Hi,
We are having this issue...Is there any workaround for this?

Thanks and regards
Maneevannan Rajkummar

**Comment by victorthoang** [comment]
Hello @Suresh-BabuLV,

Just wanted to follow up that we are actively pursuing this bug and have additional resources on the matter. I just had some clarifying questions from our dev team if you could help respond:

- Has your team tried using the latest version of the runtime to see if the regression still exists? (106)

- Does the issue still reproduce outside of kiosk mode? We've had some strange results on windows 11 and windows 10 surface tablets and just wanted to confirm.

- Could you validate if our sample app works on your devices in the win32 environment? We'd be interested to know if it's something related to an issue with the UWP framework, or if it's apparent across others.

- Is there a requirement to be using the app in UWP? As you might know, UWP is only supported as maintenance mode as we continue to build on win32 and WinUI3. It might be in our best interest to switch if there are no hard requirements here.

Thanks for your continued patience,
Victor

**Comment by tofuandeve** [comment]
Hi @Suresh-BabuLV @maneevannan After investigating this further, we have confirmed that this is not specific to WebView2 or WinUI. We were able to repro this issue on other apps with no WebView2 or any other WinUI element on kiosk mode and this seems to be a known issue with Windows Kiosk mode on older versions of Windows. However, this is fixed in later Windows versions and we have confirmed that on Windows 11 22H2, the onscreen keyboard is working in kiosk mode (for both WinUI and WebView2 edit controls). 

Closing the issue.
